cmd/imagec/storage.go

```diff
@@ -121,16 +121,14 @@ func WriteImage(image *ImageWithMeta, data io.ReadCloser) error {
 	*key = MetaDataKey
 	*blob = image.meta
 
-	r, err := client.Storage.WriteImage(
-		storage.NewWriteImageParams().
-			WithImageID(image.ID).
-			WithParentID(*image.Parent).
-			WithStoreName(image.Store).
-			WithMetadatakey(key).
-			WithMetadataval(blob).
-			WithImageFile(data).
-			WithSum(image.layer.BlobSum),
-	)
+	r, err := client.Storage.WriteImage(storage.NewWriteImageParams().
```

Comment:
This is a very minor formatting change and that's the only thing in the file.
I'm not averse to formatting improvements going in along with other changes, but when it's the only change in the file it confuses the PR and commit diff.